### ISSUE TITLE
Not working on Safari.

### ISSUE THREAD

**Issue by Eleveres** [samirkumardas/pcm-player#7]
Hi I can't manage to make your example work on safari 13.1? Do you have any idea why? The console doesn't show any error messages.
Edit: I manage to get Safari play some sound only when the sample rate is set above 25000.

**Comment by samirkumardas** [comment]
Yes, you are right, Sarfati has a buggy implementation of webAudio. 
Though minimum standard sampleRate is 8000 Hz according to the standards whereas Safari must provide at least 22050 Hz

So the provided example will not play in safari, I forgot to mention it :(

**Comment by Eleveres** [comment]
I can't manage to make it work on any mobile browser either :/

**Comment by samirkumardas** [comment]
It should work https://caniuse.com/#search=web%20audio

Note: it does not initiate playing without a user gesture

**Comment by rinkeshpanwar** [comment]
Ii have pcm player in js which takes input from websocket and play it......locally it was working fine but when i deploy to firebase it didnt work instead it works well in chrome...i dont know the problem.
